content/en/bits_ai/managing_incidents.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Managing Incidents
+kind: guide
+disable_toc: false
+private: true
+is_beta: true
+---
+
+<div class="alert alert-warning">The following features are part of the <a href="https://www.datadoghq.com/product/incident-management/">Datadog Incident Management</a> product.</div>
+
+## Prerequisites
+
+- The "Mirror all incident Slack channel messages to its incident timeline in real-time" option must be turned on in **[Incident > Settings > Integrations][3] > Slack**. This enables Datadog to generate summaries and postmortems from Slack conversations.
+- The incident channel name must be prefixed with `#incident-`. This enables you to use the Bits AI chatbot in Slack (`@Datadog`) to ask questions about the incident or give updates on it.
+
+{{< img src="bits_ai/integration-settings.png" alt="Screenshot of Slack integration settings in Datadog" style="width:90%;">}}
+
+## Generate incident summaries and scheduled updates
+
+To generate an incident summary,
+1. Join an incident channel on Slack that is linked to the Datadog Incidents app.
+1. View the incident summary when you join. Summaries are only displayed in channels with 15 or more messages. Only you see the summary, and it does not persist across reloads, between desktop and mobile apps, or across sessions.
+
+To schedule an update,
```

Comment:
At any time, you can ask for a fresh summary by asking "@Datadog Give me a summary of this incident". You can also request for updates on a schedule by asking "@Datadog Give me an update in X minutes". If you haven't already connected your Datadog account to Slack, you'll be prompted to follow the instructions on the screen.